tests/test_quick_run_config_generator.py

```diff
@@ -328,6 +329,91 @@ def test_default_config_generation(self):
     def tearDown(self):
         patch.stopall()
 
+    def test_default_ensemble_config_generation(self):
+        """
+        Test that the default ensemble config is generated correctly
+        """
+
+        fake_config = {
+            "name": "my-model",
+            "platform": "ensemble",
+            "ensemble_scheduling": {
+                "step": [{
+                    "model_name": "preprocess"
+                }, {
+                    "model_name": "resnet50_trt"
+                }]
+            },
+            "input": [{
+                "name": "INPUT__0",
+                "dataType": "TYPE_FP32",
+                "dims": [16]
+            }],
+            "max_batch_size": 4
+        }
+
+        args = [
+            'model-analyzer', 'profile', '--model-repository', '/tmp',
+            '--config-file', '/tmp/my_config.yml'
+        ]
+
+        # yapf: disable
+        yaml_str = ("""
+            profile_models:
+                - my-model:
+                    perf_analyzer_flags:
+                        percentile: 96
+            """)
+        # yapf: enable
+
+        config = evaluate_mock_config(args, yaml_str, subcommand="profile")
+
+        with patch(
+                "model_analyzer.triton.model.model_config.ModelConfig.create_model_config_dict",
+                return_value=fake_config):
+            models = [
+                ModelProfileSpec(spec=config.profile_models[0],
+                                 config=config,
+                                 client=MagicMock(),
+                                 gpus=MagicMock())
+            ]
+
+        dims = SearchDimensions()
+        dims.add_dimensions(0, [
```

Comment:
I would expect more dimensions than this. Is this ignored for this test?  Is there any testing that the SearchConfig dimensions are set up correctly? I saw that you added the code to create it

Reply:
Yes, good catch. There should be 3 more dimensions, but that wasn't the purpose of this test. I just copy/pasted so I could create a search config.
There is no test for dimensions right now. That will come in the next story when I'm creating non-default ensemble configs.